x-pack/plugins/observability/public/pages/alerts/containers/alerts_page/alerts_page.tsx

```diff
@@ -73,6 +98,48 @@ function AlertsPage() {
     },
   ]);
 
+  async function loadRuleStats() {
+    setRuleStatsLoading(true);
+    try {
+      const response = await loadRuleAggregations({
+        http,
+      });
+      // Note that the API uses the semantics of 'alerts' instead of 'rules'
+      const { alertExecutionStatus, ruleMutedStatus, ruleEnabledStatus } = response;
+      if (alertExecutionStatus && ruleMutedStatus && ruleEnabledStatus) {
+        const total = Object.entries(alertExecutionStatus).reduce((acc, [key, value]) => {
+          if (key !== 'error') {
```

Comment:
@claudiopro I am curious to know why we exclude error from total number of rules. I found the possible status values `const AlertExecutionStatusValues = ['ok', 'active', 'error', 'pending', 'unknown'] as const;`. At first glance I would expect the total number of rules to be the sum of all status values (including the errors). No?

Reply:
I wasn't sure about that, but now that you mention it, it makes sense! Let me fix that real quick in a followup PR.